v2/commands/job/logs.go

```diff
@@ -0,0 +1,133 @@
+package job
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log/slog"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	oapi "github.com/rescale-labs/htc-cli/v2/api/_oas"
+	"github.com/rescale-labs/htc-cli/v2/common"
+	"github.com/rescale-labs/htc-cli/v2/config"
+)
+
+func logs(ctx context.Context, c oapi.JobInvoker, projectId, taskId, jobId, pageIndex string) (*oapi.HTCJobLogs, error) {
+	res, err := c.GetLogs(ctx, oapi.GetLogsParams{
+		ProjectId: projectId,
+		TaskId:    taskId,
+		JobId:     jobId,
+		PageSize:  oapi.NewOptInt32(pageSize),
+		PageIndex: oapi.OptString{pageIndex, pageIndex != ""},
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	switch res := res.(type) {
+	case *oapi.HTCJobLogs:
+		return res, nil
+	case *oapi.GetLogsUnauthorized,
+		*oapi.GetLogsForbidden:
+		return nil, fmt.Errorf("forbidden: %s", res)
+	}
+	return nil, fmt.Errorf("Unknown response type: %T", res)
+}
+
+func Logs(cmd *cobra.Command, args []string) error {
+	runner, err := common.NewRunnerWithToken(cmd, time.Now())
+	if err != nil {
+		return err
+	}
+
+	p := common.IDParams{RequireProjectId: true, RequireTaskId: true}
+	if err := runner.GetIds(&p); err != nil {
+		return err
+	}
+
+	if len(args) != 1 {
+		return fmt.Errorf("Error: job ID not provided")
```

Comment:
You will want to use a `UsageError` so that the usage string gets printed to the user informing them of how to correct the problem.

```suggestion
		return config.UsageErrorf("Error: job ID not provided")
```